telas/telaAviao.py

```diff
@@ -0,0 +1,81 @@
+from telas.abstractTela import AbstractTela
+
+
+class TelaAviao(AbstractTela):
+    def __init__(self):
+        super().__init__(
+            {
+                1: "Incluir Avião",
+                2: "Alterar Avião",
+                3: "Listar Aviões",
+                4: "Excluir Avião",
+                0: "Voltar",
+            }
+        )
+
+    def tela_opcoes(self):
+        print("-------- Aviões ----------")
+        for index, opcao in self.opcoes.items():
+            print(f"{index} - {opcao}")
+        return self.verifica_opcao("Escolha uma opção: ")
+
+    def pega_dados_aviao(self):
+        print("-------- DADOS AVIÃO ----------")
+        while True:
+            modelo = input("Modelo: ").upper().strip()
+            if modelo:
+                break
+            else:
+                print("Dado invalido, digite novamente!!!")
+
+        while True:
+            fileiras = input("Fileiras: ").strip()
+            if fileiras:
+                try:
+                    fileiras = int(fileiras)
+                    break
+                except ValueError:
+                    print("Dado invalido, digite novamente!!!")
+            else:
+                print("Dado invalido, digite novamente!!!")
+        while True:
+            assentos_por_fileira = input("Assentos por fileira: ").strip()
+            if assentos_por_fileira:
+                try:
+                    assentos_por_fileira = int(assentos_por_fileira)
+                    break
+                except ValueError:
+                    print("Dado invalido, digite novamente!!!")
+            else:
+                print("Dado invalido, digite novamente!!!")
+
+        return {
+            "modelo": modelo,
+            "fileiras": fileiras,
+            "assentos_por_fileira": assentos_por_fileira,
+        }
+
+    def mostra_aviao(self, dados_aviao):
+        modelo = dados_aviao["modelo"]
+        fileiras = dados_aviao["fileiras"]
+        assentos_por_fileira = dados_aviao["assentos_por_fileira"]
```

Comment:
Poderíamos usar um `for loop` pra iterar no dicionário, assim nao precisa ficar escrevendo print por print, por exemplo:

```python
for chave, valor in dados_aviao.items():
    print(f"{chave}: {valor}")

```

como a chave do dicionário é uma string, voce pode escrever qualquer coisa, tipo: `"Modelo do Aviao"`, `Capacidade total` e etc...